tests/quick_tests/CMakeLists.txt

```diff
@@ -115,6 +115,7 @@ ENDIF()
 ADD_CUSTOM_TARGET(test
   COMMAND ${CMAKE_COMMAND} -D ALL_TESTS="${ALL_TESTS}" -P ${CMAKE_CURRENT_SOURCE_DIR}/run.cmake
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+  COMMENT "running quicktests..."
```

Comment:
We're not always consistent, but most of the time we print this as a regular sentence starting with a capital letter:
```
cmake/setup_custom_targets.cmake:      COMMENT "Build and install component \"library\"."
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:      COMMENT "Run ${TARGET} with ${CMAKE_BUILD_TYPE} configuration"
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:        COMMENT "Digitally signing ${TARGET}"
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:        COMMENT "Digitally signing ${TARGET}"
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:    COMMENT "Switch CMAKE_BUILD_TYPE to Debug"
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:    COMMENT "Switch CMAKE_BUILD_TYPE to Release"
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:    COMMENT "runclean invoked"
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:    COMMENT "distclean invoked"
cmake/macros/macro_deal_ii_invoke_autopilot.cmake:      COMMENT "strip comments"
doc/doxygen/CMakeLists.txt:  COMMENT "Generating documentation via doxygen."
doc/doxygen/options.dox.in:STRIP_CODE_COMMENTS    = YES
doc/users/cmakelists.html:  COMMENT "Switch CMAKE_BUILD_TYPE to Debug"
doc/users/cmakelists.html:  COMMENT "Switch CMAKE_BUILD_TYPE to Release"
doc/users/cmakelists.html:  COMMENT "Run with ${CMAKE_BUILD_TYPE} configuration"
tests/CMakeLists.txt:    COMMENT "Processing tests/${_category}"
tests/CMakeLists.txt:    COMMENT "Processing tests/${_category}"
```
So please capitalize and merge yourself.